File: desktop/ZlinksPackageSystem.Desktop/Models/Game.cs
using System;

namespace ZlinksPackageSystem.Desktop.Models
{
    public class Game
    {
        /// <summary>
        /// 游戏 ID(与后端 Java Long 对齐,避免 > 2^31 时的精度丢失)
        /// </summary>
        public long Id { get; set; }
        public string GameName { get; set; } = string.Empty;
        public string GameDirection { get; set; } = string.Empty;
        public string Source { get; set; } = string.Empty;
        public string GitUrl { get; set; } = string.Empty;
        public int Priority { get; set; }
        public string Tags { get; set; } = string.Empty;
        public string ProjectType { get; set; } = string.Empty;
        public string Manager { get; set; } = string.Empty;
        public string WhiteBranch { get; set; } = string.Empty;
        public string Status { get; set; } = string.Empty;
        public string RetentionRecord { get; set; } = string.Empty;
        public string AndroidFolderName { get; set; } = string.Empty;
        public string Remark { get; set; } = string.Empty;
        public DateTime CreateTime { get; set; }
        public DateTime UpdateTime { get; set; }
    }
}

File: backend/src/main/java/com/zlinks/package_system/entity/Game.java
package com.zlinks.package_system.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
@TableName("game")
public class Game extends BaseEntity {


    /** 兼容字段 - 重新声明 id 为 PK */
    @TableId(value = "id", type = IdType.AUTO)
    private Long id;
    private String gameName;
    private String gameDirection;
    private String source;
    private String gitUrl;
    private Integer priority;
    private String tags;
    private String projectType;
    private String manager;
    private String whiteBranch;
    private String status;
    private String retentionRecord;
    private String androidFolderName;
    private String remark;
}